plugins/newsdownloader.koplugin/main.lua

```diff
@@ -49,6 +49,11 @@ function NewsDownloader:addToMainMenu(menu_items)
         end
 
         FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
+        if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
+            logger.dbg("NewsDownloader: Creating init configuration")
+            FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
+                             FEED_CONFIG_PATH)
+        end
```

Comment:
Those lines seems to be merged incorrectly. Will have to remove them next commit.